### DISCUSSION TITLE
Rust game server keeps restarting with message in console "Main game process exited with code 134"

### DISCUSSION THREAD

**Discussion by Zybyr** [pterodactyl/panel#5066] [Support]
Just installed pterodactyl and trying to run Rust game server. Right after the server starts there is a message in console `Main game process exited with code 134` and the server starts rebooting. 

Pterodactyl is installed in RHEL9 physical computer with 64GB DDR5 memory. 

Any help please?

**Comment by matricali** [comment]
@Kn1az Were you able to solve it? I have the same problem
